lib/configure.js

```diff
@@ -272,6 +272,7 @@ function configure (gyp, argv, callback) {
       argv.push('-Dnode_gyp_dir=' + nodeGypDir)
       argv.push('-Dnode_lib_file=' + release.name + '.lib')
       argv.push('-Dmodule_root_dir=' + process.cwd())
+      argv.push('-Dnode_engine=' + (gyp.opts.node_engine || process.jsEngine || 'v8'))
```

Comment:
@bnoordhuis , sorry I didn't exactly follow your comment. Could you please elaborate? 
`process.jsEngine` is something that was added in `node-chakracore` to distinguish between engines.